keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -710,6 +781,43 @@ intended.
 
 The atomic resize requirement should be reevaluated prior to GA, and in the context of pod-level resources.
 
+### Actuating Resizes
+
+The resources specified by the Kubelet are not guaranteed to be the actual resources configured for
+a pod or container. Examples include:
+- Linux kernel enforced minimums for CPU shares & quota
+- Systemd cgroup driver rounds CPU quota up to the nearest 10ms
+- NRI plugins can change resource configuration
+
+Therefore the Kubelet cannot reliably compare desired & actual resources to know whether to trigger
+a resize (a level-triggered approach).
+
+To accommodate this, the Kubelet stores the set of "acknowledged" resources per container.
+Acknowledged resources represent the resource configuration that was passed to the runtime (either
+via a CreateContainer or UpdateContainerResources call) and received a successful response. The
+acknowledged resources are checkpointed alongside the allocated resources to persist across
+restarts. There is the possibility that a poorly timed restart could lead to a resize request being
+repeated, so `UpdateContainerResources` must be idempotent.
+
+When a resize CRI request succeeds, the pod will be marked for resync to read the latest resources. If
```

Comment:
Worth adding a section on exploring whether to activel re-assert periodically and at start?  I think we should demand that re-asserting be a low-impact operation in the already-correct case (make that the CRI's problem) and just not be worried about re-asserting too often.

Reply:
I think this is worth exploring :+1: